lib/foreman/renderer/configuration.rb

```diff
@@ -42,7 +42,8 @@ class Configuration
         :number_with_delimiter,
         :number_with_precision,
         :number_to_human_size,
-        :gem_version_compare
+        :gem_version_compare,
+        :foreman_ssl_certificate
```

Comment:
Let's use a trailing comma to having to modify it in the future when another line is added.
```suggestion
        :foreman_ssl_certificate,
```

Reply:
Turns out Rubocop dislikes this. We should solve that elsewhere.